### PR TITLE
Titouan

### PR DESCRIPTION
**Pull request by Resident-png**
Pretty much finished the sign up page. The only thing left to change (if we want to) is to make sure the email adress isn't already in the database. We might also want to actually check if the password and confirm password are the same.

### CODE REVIEW

**Comment by Resident-png** [comment]
Pull request for work on issues #3 and #72.

**Review by shakhal350 — APPROVED** [review]

File: src/Components/SignUp.js [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('');
-    const [user, setUser] = useState('');
+    const [confirmPassword, setConPassword] = useState('');
+    const [userType, setUserType] = useState('');
+    
+    const userTypes = ["", "Customer", "Seller", "Admin"];
+    const [error, setError] = useState(false);
 
     const handleChangeFN = (event) => {
```
Comment (tenzabest) [inline src/Components/SignUp.js:25]: Change function name
Ex : handleChangeFN should be handleChangeFirstName

File: src/Components/SignUp.js [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 import InputAdornment from "@mui/material/InputAdornment";
-import Box from '@mui/material/Box';
 import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
 import FormControl from '@mui/material/FormControl';
 import Select from '@mui/material/Select';
+import { SignUpUser } from "../firebaseService";
 
 const SignUp = () =>  {
     const [firstname, setFirstName] = useState('');
```
Comment (tenzabest) [inline src/Components/SignUp.js:15]: firstName

File: src/Pages/LoginPage.js [hunk 3]
```diff
@@ -9,7 +9,6 @@ import LockIcon from "@mui/icons-material/Lock";
 import InputAdornment from "@mui/material/InputAdornment";
 import React, { useState } from "react";
 import { SignInUser } from "../firebaseService";
-import SignUp from "../Components/SignUp";
```
Comment (tenzabest) [inline src/Pages/LoginPage.js:12]: Remove import (Not used)

**Review by tenzabest — CHANGES_REQUESTED** [review]
Before merging, make sure to :
-Compare password with confirm password
-Verify that all text fields are completed before sign up 
-Verify if email used to sign up doesn't exist 

Pull request should be done in the working branch
Make sure you have a descriptive commit name

**Comment by Resident-png** [comment]
I will close this and do another PR to the working branch

**Pull request closed without merging**